appveyor.yml

```diff
@@ -3,21 +3,19 @@ version: '{build}'
 skip_tags: true
 
 environment:
+  JRUBY_OPTS: "--dev -J-Xmx1024M --debug"
   matrix:
-    - ruby_version: "21"
-    - ruby_version: "21-x64"
-    - ruby_version: "jruby-9.0.4.0"
+    - ruby_version: "Ruby21"
+    - ruby_version: "Ruby21-x64"
+    - ruby_version: "jruby-9.0.0.0"
 
 cache:
   - vendor/bundle
 
 install:
-  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
-  - ruby --version
-  - gem --version
+  - SET PATH=C:\%ruby_version%\bin;%PATH%
   - gem install bundler
-  - bundler --version
-  - bundle platform
+  - bundle env
```

Comment:
I'm guessing `bundle env` is sufficient for ruby --version, gem --version bundler --version and bundle platforM?